src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_horizontal_layout.html.twig

```diff
@@ -76,6 +74,10 @@ col-sm-2
 {% endspaceless %}
 {% endblock reset_row %}
 
-{% block form_group_class -%}
-col-sm-10
-{%- endblock form_group_class %}
+{% block form_group_class %}
+    {%- if label is same as(false) -%}
+        col-sm-12
+    {%- else -%}
+        col-sm-10
+    {%- endif %}
```

Comment:
No, don't use ``{% spaceless %}``:

- it removes spaces between HTML tags only, so it won't work at all here
- it removes spaces at runtime with a regex, so it makes the rendering of form slower than using whitespace control (which is applied during the parsing of the template)